pkg/monitoring/vms/prometheus/prometheus.go

```diff
@@ -224,6 +224,11 @@ func (metrics *vmiMetrics) updateBlock(blkStats []stats.DomainStatsBlock) {
 			continue
 		}
 
+		diskName := block.Name
+		if block.AliasSet {
+			diskName = block.Alias
```

Comment:
We have some metrics e2e tests. can you add a check there if the aliases pick up?